aws/windows-agent/main.tf

```diff
@@ -29,12 +29,26 @@ module "dcos" {
   bootstrap_instance_type = "m4.xlarge"
 
   dcos_variant              = "ee"
-  dcos_version              = "1.13.1"
+  dcos_version              = "2.1.0"
   dcos_license_key_contents = "${file("~/license.txt")}"
+  ansible_bundled_container = "mesosphere/dcos-ansible-bundle:windows-beta-support"
+
+  #Linux Installer path - place url with "pull/PR#" or "master" suffix here:
+  custom_dcos_download_path = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
 
   # provide a SHA512 hashed password, here "deleteme"
   dcos_superuser_password_hash = "$6$rounds=656000$YSvuFmasQDXheddh$TpYlCxNHF6PbsGkjlK99Pwxg7D0mgWJ.y0hE2JKoa61wHx.1wtxTAHVRHfsJU9zzHWDoE08wpdtToHimNR9FJ/"
   dcos_superuser_username      = "demo-super"
+
+  dcos_config = <<-EOF
+enable_windows_agents: true
+-EOF
+
+  #Windows Installer path - place url with "pull/PR#" or "master" suffix here:
+ansible_additional_config = <<-EOF
+dcos:
+ download_win: "https://downloads.mesosphere.com/dcos-enterprise/testing/master/windows/dcos_generate_config_win.ee.sh"
```

Comment:
I see the point here but for a tested version of this example with a stable result an immutable release should be chosen. E.g. a commit-hash which was verified to work with this example. 

If this stay with master results will be different every time this example is ran by someone

Reply:
for now master artifact is good enough.